tests/python/gaia-ui-tests/gaiatest/visual_components/switch.py

```diff
@@ -0,0 +1,18 @@
+from marionette_driver import Wait
+from gaiatest.visual_components.visual_component import VisualComponent
+
+
+class Switch(VisualComponent):
```

Comment:
So Switch/HtmlSwitch is the input switch and GaiaSwitch is gaia-switch?
This isn't part of this pull request, right? Could you file a new bug on this?